client/data/payment-activity/test/reducer.test.ts

```diff
@@ -12,6 +12,11 @@ describe( 'receivePaymentActivity', () => {
 		fees: 300,
 		disputes: 315,
 		refunds: 200,
+		currency: 'JPY',
```

Comment:
Shouldn't the value of currency in lower case?

I see in some other places as well in this PR's diff.

It's not breaking anything because any use of the currency code in `formatCurrency()` converts the string to upper case or lower case depending on the need correctly, so passing currency code in lower or upper case to `formatCurrency()` will work fine.

It's just that the response from server will always be in lower case.

Reply:
Interestingly `wcpaySettings.currencyData` does use uppercase. But I've adjusted others to lowercase where applicable in 0c74c13, d1630b7.

https://github.com/Automattic/woocommerce-payments/blob/0c74c13a0c014f4417e094b84cc1faa982dee625/client/components/payment-activity/test/payment-data-tile.test.tsx#L30-L32